default-menu-auto-hide.sh

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (C) 2022  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Jiri Konecny <jkonecny@redhat.com>
+
+# We have this feature on Fedora, however, only for Workstation which is not
+# used by KS tests. There is another test which works for Fedora only and
+# changing configuration file.
+TESTTYPE="dracut skip-on-rhel-8 skip-on-fedora"
```

Comment:
Is this test really about dracut? Isn't it about the bootloader configuration?

Reply:
Do we have such category now?

Reply:
Yes, it is `bootloader`.

Reply:
Used group `bootloader` instead.